src/accelerate/inference.py

```diff
@@ -111,9 +115,10 @@ def prepare_pippy(
     example_args=(),
     example_kwargs: Optional[Dict[str, Any]] = None,
     num_chunks=None,
+    gather_output=False,
```

Comment:
Based on the documentation above : `When finished all the data will be on each GPU for convenience:`

```suggestion
    example_args=(),
    example_kwargs: Optional[Dict[str, Any]] = None,
    num_chunks=None,
    gather_output=True,
```

Reply:
Updating docs to reflect `False`, as per discussion with Ke we want to make this not the default.